admin/app/components/solidus_admin/orders/show/component.rb

```diff
@@ -10,4 +10,51 @@ def initialize(order:)
   def form_id
     @form_id ||= "#{stimulus_id}--form-#{@order.id}"
   end
+
+  def format_address(address)
+    return unless address
+    safe_join([
+      address.name,
+      tag.br,
+      address.address1,
+      tag.br,
+      address.address2,
+      address.city,
+      address.zipcode,
+      address.state.name,
+      tag.br,
+      address.country.name,
+      tag.br,
+      address.phone,
+    ], " ")
+  end
+
+  def panel_title_with_more_links(title, links)
+    tag.details(
+      tag.summary(
+        tag.div(
+          safe_join([
+            title,
+            component("ui/button").new(
+              icon: "more-line",
+              scheme: :ghost,
+              tag: :div,
+              alt: t("spree.edit"),
+              class: "cursor-pointer"
+            ).render_in(self),
+          ]),
+          class: 'flex items-center justify-between text-black',
+        )
+      ) + tag.div(safe_join(links, " "), class: "body-small absolute border border-gray-100 mt-0.5 right-0 flex min-w-[10rem] flex-col p-2 rounded-sm shadow-lg bg-white z-10"),
+      class: 'relative',
+    )
+  end
+
+  def customer_name(user)
+    (user.default_user_bill_address ||
+      user.default_user_ship_address ||
+      user.user_addresses.where(default: true).first ||
+      user.user_addresses.first ||
+      Spree::Address.new).name
+  end
```

Comment:
The `name` is expected to be called on a `Spree::Address` object, but when the condition falls through to a `user.user_addresses` collection, the `name` is being called on a `Spree::UserAddress` object instead, which doesn't have a name defined on it: https://github.com/solidusio/solidus/pull/5461/commits/671edabba9aecc64ee079009aae726d2e5cd76a7

Reply:
Thanks! Fixed 👍